Runtime/Tracking/Velocity/ComponentTrackerProxy.cs

```diff
@@ -46,6 +51,14 @@ protected override Vector3 DoGetAngularVelocity()
         /// </summary>
         [CalledAfterChangeOf(nameof(ProxySource))]
         protected virtual void OnAfterProxySourceChange()
+        {
+            SetCachedVelocityTracker();
+        }
+
+        /// <summary>
+        /// Sets the VelocityTracker cache to the first found <see ref="VelocityTracker"> on the <see ref="ProxySource"> or any of its descendants.
+        /// </summary>
+        protected virtual void SetCachedVelocityTracker()
         {
             cachedVelocityTracker = ProxySource == null ? null : ProxySource.GetComponentInChildren<VelocityTracker>();

```

Comment:
We have a nice extension method for this:
```suggestion
            cachedVelocityTracker = ProxySource.TryGetComponent<VelocityTracker>(true);
```

Reply:
done in new commit